FinalChallenge/Controllers/AllTasks/AllTasksViewModel.swift

```diff
@@ -33,8 +33,7 @@ class AllTasksViewModel<FrequentRepository: RepositoryErasurer<FrequencyTask, NS
     func readAllFrequencyTasks() {
         frequentRepository.readAll { [weak self] result in
             switch result {
-            case .error(let error):
-                print(error)
+            case .error:
                 fatalError("error no frenquent repository")
```

Comment:
change frenquent to frequent
```suggestion
                fatalError("error no frequent repository")
```

Reply:
done